test/sw_mgmt_spec.rb

```diff
@@ -178,8 +178,8 @@
       expect(File).to receive(:exist?).with(File.join(root_dir, target_dir, "SCCcredentials"))
         .and_return(false)
 
-      expect(FileUtils).to receive(:cp).with(File.join(root_dir, target_dir, "NCCcredentials"),
-        File.join(target_dir, "SCCcredentials"))
+      expect(subject).to receive(:`).with("cp -a " + File.join(root_dir, target_dir,
+        "NCCcredentials") + " " + File.join(target_dir, "SCCcredentials"))
```

Comment:
i assume that it will work correctly since it worked before, but still for my cusiousity: Why is root_dir joined only for the source file?